Python/sysmodule.c

```diff
@@ -336,6 +352,23 @@ PySys_Audit(const char *event, const char *argFormat, ...)
     return res;
 }
 
+int
+PySys_AuditTuple(const char *event, PyObject *args)
+{
+    if (args) {
```

Comment:
Nitpick: the code may be clearer (at leas you can get rid of one indentation level) if handle the case for `args == NULL` first. It is only a matter of personal preferences.